pkg/target/target.go

```diff
@@ -337,12 +344,13 @@ func (h *K8sValidationTarget) MatchSchema() apiextensions.JSONSchemaProps {
 					},
 				},
 			},
-			"namespaces":         wildcardNSList,
-			"excludedNamespaces": wildcardNSList,
-			"labelSelector":      labelSelectorSchema,
-			"namespaceSelector":  labelSelectorSchema,
+			"namespaces":         propsWithDescription(wildcardNSList, "namespaces is a list of namespace names. If defined, a constraint will only apply to resources in a listed namespace."),
+			"excludedNamespaces": propsWithDescription(wildcardNSList, "excludedNamespaces is a list of namespace names. If defined, a constraint will only apply to resources not in a listed namespace."),
+			"labelSelector":      propsWithDescription(labelSelectorSchema, "labelSelector is a standard Kubernetes label selector."),
+			"namespaceSelector":  propsWithDescription(labelSelectorSchema, "namespaceSelector is a standard Kubernetes namespace selector. If defined, make sure to add Namespaces to your configs.config.gatekeeper.sh object to ensure namespaces are synced into OPA."),
```

Comment:
"standard K8s namespace selector" I'm not sure that's a thing. It's a "label selector against the containing namespace, or against the namespace itself"

Also, I don't think we require namespaces to be synced any more, unless the user is auditing from cache (which is not the default).

Reply:
I've updated with a paraphrase of your comment. PTAL